torax/_src/imas_tools/input/core_profiles.py

```diff
@@ -109,3 +110,110 @@ def profile_conditions_from_IMAS(
       "normalize_n_e_to_nbar": False,
       "v_loop_lcfs": v_loop_lcfs,
   }
+
+
+def plasma_composition_from_imas(
```

Comment:
As a general point for these loaders as IMAS doesn't give us any guarantees on what data is filled in I'm leaning towards doing some input validation. wdyt?